.github/workflows/publish.yaml

```diff
@@ -13,6 +14,7 @@ jobs:
     uses: ./.github/workflows/ci.yaml # use the callable ci job to run CI checks
 
   publish-update-bundle:
+    if: github.head_ref == 'preview'
```

Comment:
thoughts?: only create expo view for preview, since anything that would be released would go through preview/test flight first anyways, no need to have redundant expo views.

Reply:
Isn't the `bundle` here the JavaScript bundle we're shipping OTA for everyone? Seems to me like we should always build it, but based on the branch we're building from, we put it into different branches.

Reply:
in theory, yes, and what I am saying for now, until we understand how the bundle works better, lets not update it in prod and only push updates via builds/new versions. 

going to read about bundles today.

Reply:
Ah, gotcha - if we do that, we will have a day- to days-long lag between new builds we publish and users getting them. If we are diligent about pushing bundles to the right branch (so that e.g. folks on TestFlight only get things we intend for them to) I think we can reduce app store review headache by using them.

Reply:
wouldn't you want to build bundles for `main` as well? I'm thinking that someone like a product owner (Charlotte) should be able to open Expo Go and see work that's been recently completed on main, and not have to wait until the point of prepping a new preview branch.

Reply:
yes, great point Brian! I think between the channel for main being called preview today I got confused :D